src/Microsoft.Graph/Generated/callrecords/requests/CallRecordSessionsCollectionResponse.cs

```diff
@@ -9,25 +9,23 @@
 namespace Microsoft.Graph.CallRecords
 {
     using System.Collections.Generic;
-    using System.Runtime.Serialization;
-    using Newtonsoft.Json;
+    using System.Text.Json.Serialization;
 
     /// <summary>
     /// The type CallRecordSessionsCollectionResponse.
     /// </summary>
-    [JsonObject(MemberSerialization = MemberSerialization.OptIn)]
     public class CallRecordSessionsCollectionResponse
     {
         /// <summary>
         /// Gets or sets the <see cref="ICallRecordSessionsCollectionPage"/> value.
         /// </summary>
-		[JsonProperty(NullValueHandling = NullValueHandling.Ignore, PropertyName ="value", Required = Newtonsoft.Json.Required.Default)]
+		[JsonPropertyName("value")]
```

Comment:
nit: indent is off

Reply:
https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/870/commits/01fa5093689721cf5d6cc91b92bd04c443d5ae4b Resolves indentation issues.